src/apps/schema/src/app/utils/index.ts

```diff
@@ -19,6 +19,15 @@ export const modelNameMap = {
   block: "Block",
 };
 
+export const RESERVED_FIELD_NAMES = [
+  "og_image",
+  "og_title",
+  "og_description",
+  "tc_title",
+  "tc_description",
+  "tc_image",
```

Comment:
@geodem127 if I remember correctly, the following field names **should** be allowed for manual creation as these fields do not get automatically created like the `og_image` field. We only want `og_image` field to be blocked from manual creation, the others should still be allowed as we have users manually creating these for their models since there's no other way to generate those fields.